### DISCUSSION TITLE
Are NSFW words available?

### DISCUSSION THREAD

**Discussion by Mikaru0Mystic** [mcnaveen/Random-Words-API#84] [Q&A]
I was wondering if NSFW words, pharases, are in the lists used. 
I'm hoping to use this but couldn't do it if they are :)

**Comment by mcnaveen** [comment]
We are scrapping words from third-party website. So I can't guarantee there won't be any. However, we are planning to host the English words ourselves. Hope it'll filter out NSFW Words.
